Gemfile

```diff
@@ -35,18 +35,20 @@ group :development do
   gem 'travis',       :require => false
   gem 'travis-lint',  :require => false
   gem 'guard-rake',   :require => false
+  gem 'simplecov',    :require => false
+  gem 'listen', '<= 3.0.6', :require => false
 end
 
 group :system_tests do
-  gem 'beaker',                        :require => false
-  if beaker_version = ENV['BEAKER_VERSION']
-    gem 'beaker', *location_for(beaker_version)
-  end
-  if beaker_rspec_version = ENV['BEAKER_RSPEC_VERSION']
-    gem 'beaker-rspec', *location_for(beaker_rspec_version)
-  else
-    gem 'beaker-rspec',  :require => false
-  end
+  gem 'winrm'
```

Comment:
this block will be killed with the next modulesync, could you also add the gems to .sync.yml?